src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -172,43 +174,47 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       }
     ];
 
+    const currentPermType = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
+
     if (!panel) {
       columns.push(
         {
           key: 'consentDescription',
           name: messages.Description,
           fieldName: 'consentDescription',
           isResizable: true,
-          minWidth: (tokenPresent) ? 400 : 600,
-          maxWidth: (tokenPresent) ? 600 : 1000,
+          minWidth: (tokenPresent) ? ((currentPermType) ? 400 : 800) : 600,
+          maxWidth: (tokenPresent) ? ((currentPermType) ? 600 : 1300) : 1000,
```

Comment:
i dont think u need all the parenthesis. you can just do
`tokenPresent ? (currentPermType ? 400 : 800) : 600,`

Reply:
I agree, but the original code had parenthesis around just `tokenPresent`, so what I wrote is technically consistent with what they had before. However, if you think this sacrifices too much clarity, I can make the change.